Watchdog Setup Notification Function

JKTUNING opened the pull request:
Checks config.js file for watchdog_setup = 1

If setup is true then send discord notification and write watchdog_setup = 0 in config.js

XK4MiLX commented:
Code not working
- await is only valid in async functions and the top level bodies of modules
before any PR pls test your solution

JKTUNING commented:
Apologies. Will make to appropriate change and update. Good catch!

JKTUNING commented:
Added code to a live node to test. Removed async and await from function definition and calls.

Fixed error in variable checking and formatting.

Added console log for confirmation message.

![watchdog_setup ping](https://user-images.githubusercontent.com/26805518/202440852-945eb2d6-f6e9-47ca-806d-573e9f7cce86.PNG)
